operator/flags.go

```diff
@@ -298,5 +298,15 @@ func init() {
 	flags.Bool(option.SkipCRDCreation, false, "When true, Kubernetes Custom Resource Definitions will not be created")
 	option.BindEnv(option.SkipCRDCreation)
 
+	flags.Bool(option.EnableCiliumEndpointSlice, false, "If set to true, the CiliumEndpointSlice feature is enabled. If any CiliumEndpoints resources are created, updated, or deleted in the cluster, all those changes are broadcast as CiliumEndpointSlice updates to all of the Cilium agents.")
+	option.BindEnv(option.EnableCiliumEndpointSlice)
+
+	flags.Int(operatorOption.CESMaxCEPsInCES, operatorOption.CESMaxCEPsInCESDefault, "Maximum number of CiliumEndpoints allowed in a CES")
+	option.BindEnv(operatorOption.CESMaxCEPsInCES)
+
+	flags.String(operatorOption.CESSlicingMode, operatorOption.CESSlicingModeDefault, "Slicing mode define how ceps are grouped into a CES")
+	flags.MarkHidden(operatorOption.CESSlicingMode)
+	option.BindEnv(operatorOption.CESSlicingMode)
```

Comment:
Do you expect that users may want to configure these other options via Helm?

If that's the case, then we would typically define the helm option as a nested configuration with the various options:

```
CiliumEndpointSlice:
  enabled: true
  maxEndpointsInSlice: n
  mode: foo
```

Reply:
good point, I don't think we would want to expose the internal params of CES to users, they should only see an enable option in the config.

Reply:
Ah right, I missed that these already had `MarkHidden()`. In that case I think that no changes should be necessary to account for helm configuration (unless we do expect to expose configuration args to users in future).

Reply:
^^ Correction, CESSlicing mode is marked hidden but CESMaxCEPsInCES is _not_ marked hidden today so do we want to expose that one?

Reply:
I had  a offline conversation with team regarding `CESMaxCEPsInCES`, we decided to keep this config parameter also hidden.